tests/integration/cli/repos_test.py

```diff
@@ -24,19 +24,19 @@ def setUp(self):
         os.chmod(self.fake_override_dir, 0777)
         os.chmod(self.fake_from_dir, 0777)
 
-        self.old_repos_dir = constants.REPOS_DIR
-        constants.REPOS_DIR = self.temp_repos_dir
         self.fake_override_repo_location = os.path.join(self.fake_override_dir, 'fake-repo')
         self._set_up_fake_local_repo(path=self.fake_override_repo_location)
         self.fake_from_repo_location = os.path.join(self.fake_from_dir, 'fake-repo')
         self._set_up_fake_local_repo(path=self.fake_from_repo_location)
+        self.fake_source_repo_location = '/tmp/fake-repo'
```

Comment:
is there a reason why we are stuck using /tmp instead of mkdtemp ? I seem to recall there being something why